server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

```diff
@@ -333,8 +333,10 @@ This redirect usually happens when the user clicks the `Log Out` link on the pag
 
 Once the user is redirected to the logout endpoint, {project_name} is going to send logout requests to
 clients to let them to invalidate their local user sessions, and potentially redirect the user to some URL
-once the logout process is finished. The user might be optionally requested to confirm the logout in case that `id_token_hint` parameter was not used
-or the client referenced in the ID Token from `id_token_hint` has `Consent Required` enabled.
+once the logout process is finished. The user might be optionally requested to confirm the logout in case that `id_token_hint` parameter was not used.
```

Comment:
```suggestion
once the logout process is finished. The user might be optionally requested to confirm the logout in case the  `id_token_hint` parameter was not used.
```